backends/cuda/ceed-cuda-operator.c

```diff
@@ -61,6 +61,10 @@ static int CeedOperatorSetupFields_Cuda(CeedQFunction qf, CeedOperator op,
   CeedElemRestriction Erestrict;
   CeedOperatorField *opfields;
   CeedQFunctionField *qffields;
+  CeedVector field_vec;
+  bool is_strided;
+  bool skip_restrict;
```

Comment:
Similar naming convention comment. Personally, I'm a fan of `camelCase` and its in the CPU backends. I don't think we use `snake_case` anywhere but the Python code, except for using `d_foo` and `h_foo`.

Reply:
MAGMA itself uses `snake_case`, but you're right that I should match the CUDA backend style in this file.  I changed `is_strided` to `strided`, `skip_restrict` to `skiprestrict`, and `field_vec` to `fieldvec`, which I think "blends in" more.